util/include/Timers.hpp

```diff
@@ -0,0 +1,144 @@
+// Concord
+//
+// Copyright (c) 2018 VMware, Inc. All Rights Reserved.
```

Comment:
Are you getting this header file from somewhere existing or is it new? I notice you are using a 2018 copyright here rather than a 2019 copyright.

Reply:
typo: fixed.